src/config/wayland.md

```diff
@@ -13,7 +13,7 @@ server, the window manager and the compositor in a single application.
 Both GNOME and KDE Plasma have Wayland sessions. For GNOME, Wayland is already
 the default session, while Plasma still has a few issues to iron out, so their
 default remains a Xorg session. While running these desktop environments,
-applications built with Gtk+ will automatically choose the Wayland backend, but
+applications built with GTK+ will automatically choose the Wayland back-end, but
```

Comment:
I would prefer "backend" as this is the form usually used in this context.

Reply:
I was really unsure about this one. Reverted